omniscidb/QueryEngine/JoinHashTable/Runtime/HashJoinRuntimeCpu.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+
+#include "../../../Shared/sqltypes.h"
+
+namespace cpu_utils {
```

Comment:
The namespace name is too common and kind of misleading. I'd say we don't use any namespace here at all.

Reply:
removed namespace